Package.swift

```diff
@@ -1,23 +1,28 @@
-// swift-tools-version:4.2
+// swift-tools-version:5.7
 
 import PackageDescription
 
 let package = Package(
     name: "SwiftDEBot",
+    platforms: [
+        .macOS(.v12),
+    ],
     dependencies: [
-        .package(url: "https://github.com/Azoy/Sword.git", from: "0.9.2"),
+        .package(url: "https://github.com/MahdiBM/DiscordBM", revision: "53e8bd10a4489cd5a8c2c7a47d8801e20a3df530"),
```

Comment:
I tagged the changes we were talking about so you can use `exact: "1.0.0-beta.49"` now

Reply:
Awesome, thank you!